.changeset/clean-points-glow.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-tech-radar': minor
```

Comment:
Could you make this a patch level instead? It's a nonbreaking change and those are patch level on 0.x versions.

Reply:
No problem, Done. 

Thanks for the heads up on the  0.x convention. 👍